tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyLog.java

```diff
@@ -74,6 +74,16 @@ public static void setDir(File homeFile) {
 
     public static void logException(Logger logger, final Throwable exception, String what, WebContext ctx) {
         logger.log(Level.SEVERE, what, exception);
+        countException(exception);
+    }
+
+    /**
+     * Count an exception in the SurveyMetrics
+     */
+    public static void countException(final Throwable exception) {
+        if (CookieSession.sm != null && CookieSession.sm.surveyMetrics != null) {
+            CookieSession.sm.surveyMetrics.countException(exception);
+        }
```

Comment:
jaxrs stuff ( i.e. the `.api` subpackage ) won't need this hackery, somethiing like:

```java
class whatever {
     @Inject
     SurveyMetrics surveyMetrics;
```

… will be sufficient.

Reply:
the above is only to allow calling from a static environment